content/_code-samples/escrow/py/create_escrow.py

```diff
@@ -0,0 +1,52 @@
+from xrpl.wallet import Wallet, generate_faucet_wallet
+from xrpl.clients import JsonRpcClient
+from datetime import datetime, timedelta
+from xrpl.utils import xrp_to_drops
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.utils import datetime_to_ripple_time
+from xrpl.models import EscrowCreate                             
+
+# Create Escrow
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # instanstiate ripple client
+
+amount = 10.000 # amount to escrow
```

Comment:
Instead of the comment, you can always include that clarification in the variable name if you'd like - Ex. `amount_to_escrow`, although I think`amount` on it's own is fine